docs/src/gcode/tool-compensation.adoc

```diff
@@ -175,69 +175,69 @@ is used for all machines.
 [[sub:tool-io]]
 === Tool IO(((Tool IO)))
 
-The userspace program specified by *[EMCIO]EMCIO = io* is conventionally
+The userspace program specified by `[EMCIO]EMCIO = io` is conventionally
 used for tool changer management (and other io functions for enabling
 LinuxCNC and the control of coolant/lube hardware).  The HAL pins used for
-tool management are prefixed with *iocontrol.0.*.
+tool management are prefixed with `iocontrol.0.`.
 
-A G-code *T* command asserts the HAL output pin *iocontrol.0.tool-prepare*.
-The HAL input pin, *iocontrol.0.tool-prepared*, must be set by external
+A G-code `*T*` command asserts the HAL output pin `iocontrol.0.tool-prepare`.
```

Comment:
I think this was not intended either:

![grafik](https://user-images.githubusercontent.com/67957916/193787568-9a577224-eae7-409c-a012-7fd3bf2eade2.png)

Reply:
A fat monospaced was meant.

Reply:
> A fat monospaced was meant.

And too renders nicely in AsciiDoctor, as well as *`T`* which is the form I'm favoring, selecting font type (`monospace`), then style (_italic_), then weight (*bold*), which happens to follow the order described here https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Fundamentals.

Reply:
Fine with me.

Reply:
Except that I am thinking from the outside to the inside. I will change the type less frequently than the style or weight. Bike shedding.

Reply:
Also not fixed yet